velox/functions/sparksql/tests/ElementAtTest.cpp

```diff
@@ -57,26 +57,19 @@ TEST_F(SubscriptTest, allFlavors2) {
       makeMapVector<int64_t, int64_t>(1, sizeAt, keyAt, mapValueAt);
 
   // #1
-  EXPECT_EQ(subscriptSimple("getarrayitem(C0, 0)", {arrayVector}), 10);
-  EXPECT_EQ(subscriptSimple("getarrayitem(C0, 1)", {arrayVector}), 11);
-  EXPECT_EQ(subscriptSimple("getarrayitem(C0, 2)", {arrayVector}), 12);
+  EXPECT_EQ(subscriptSimple("element_at(C0, 1)", {arrayVector}), 10);
```

Comment:
> subscriptSimple

Should we rename this to elementAtSimple? Let's check other method names.

Reply:
fixed it.